### PR TITLE
Improve error message

### PR DESCRIPTION
**Pull request by martin-schibsted**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

Improves error message for incorrect auth field. The current message is confusing/nondescriptive.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

Yes. 

```release-note
Improved error message for incorrect auth field.

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by matthyx** [comment]
/cc
@martin-schibsted please sign the CLA in order to process your PR

**Comment by martin-schibsted** [comment]
I've signed the CLA.

File: pkg/credentialprovider/config.go [hunk 1]
```diff
@@ -289,7 +289,7 @@ func decodeDockerConfigFieldAuth(field string) (username, password string, err e
 
 	parts := strings.SplitN(string(decoded), ":", 2)
 	if len(parts) != 2 {
-		err = fmt.Errorf("unable to parse auth field")
+		err = fmt.Errorf("unable to parse auth field. Expected auth field to be of the format username:password.")
```
Comment (liggitt) [inline pkg/credentialprovider/config.go]:
```suggestion
		err = fmt.Errorf("unable to parse auth field, must be formatted as base64(username:password)")
```

**Comment by matthyx** [comment]
/retest
@liggitt can you check?

**Comment by matthyx** [comment]
@liggitt why do we have so much flakes here?

**Pull request merged**